airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/commands.py

```diff
@@ -263,18 +262,3 @@ async def connectors(
         ctx.obj["enable_dependency_scanning"],
     )
     log_selected_connectors(ctx.obj["selected_connectors_with_modified_files"])
-
-
-async def get_modified_files(git_branch: str, git_revision: str, diffed_branch: str, is_local: bool, ci_context: CIContext) -> Set[str]:
```

Comment:
I moved this function in the `pipelines.helpers.git ` module as its reused in `airbyte-ci test` command for file change detection.